engine/src/main/java/com/datatorrent/stram/webapp/StramWebServices.java

```diff
@@ -527,6 +558,19 @@ public JSONObject getLogicalOperator(@PathParam("operatorName") String operatorN
   }
 
   @GET
+  @Path(PATH_LOGICAL_PLAN_MODULES + "/{moduleName}")
```

Comment:
As discussed this is not needed as the information can already be retrieved with the logical plan.